README.md

```diff
@@ -6,4 +6,44 @@ Travis:   [![Travis](https://travis-ci.org/aspnet/MvcPrecompilation.svg?branch=d
 
 The Razor syntax provides a fast, terse, clean, and lightweight way to combine server code with HTML to create dynamic web content. This repo contains tooling that allows compilation of MVC Razor views as part of build and publish.
 
+## Installation and usage
+
+### Referencing the `Microsoft.AspNetCore.Mvc.Razor.ViewCompilation` package
+* If you're targeting ASP.NET Core 2.0 or higher on CoreCLR, a reference to the `Microsoft.AspNetCore.Mvc.Razor.ViewCompilation` package is added by `Microsoft.AspNetCore.All` and you do not need to explicitly reference it.
+* For desktop targeting projects or projects targeting ASP.NET Core 1.x, add a package reference to the appropriate version of `Microsoft.AspNetCore.Mvc.Razor.ViewCompilation` in your project:
+
+```xml
+<ItemGroup>
+  <Microsoft.AspNetCore.Mvc.Razor.ViewCompilation Version="1.1.1" />
+</ItemGroup>
+```
+
+### Enabling view compilation
+To enable compilation as part of publishing, add the `MvcRazorCompileOnPublish` property:
+```xml
+<PropertyGroup>
+  <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>
+</PropertyGroup>
+```
```

Comment:
Yeah it is now. I'll add a note saying you need this if you aren't referencing the WebSdk